rag/infrastructure/http/slack.py

```
@@ -0,0 +1,172 @@
+    """
+    try:
+        connector = slack_connector(DEFAULT_PROJECT_ID)
+        result = connector.get_available_slack_channels_from_cache(
+            types=types,
+            cursor=cursor if cursor else None,
+            limit=limit,
+            search_regex=search_regex,
+        )
+        return jsonify(result.to_dict(data_key="channels")), 200
+        
+    except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)